frontend/src/views/WheresMyGene/components/HeatMap/components/YAxisChart/index.tsx

```diff
@@ -159,10 +202,44 @@ export default memo(function YAxisChart({
         )}
       </TissueWrapper>
       <Container
-        data-test-id="cell-type-labels"
+        data-test-id={`cell-type-labels-${tissueKey}`}
         height={heatmapHeight}
         ref={yAxisRef}
       />
+      {yAxisInfoCoords &&
+        yAxisInfoCoords.map((coord, i) => {
+          const content = cellTypeMetadata[i];
+          return (
+            <div
+              id={`${content}`}
+              key={`${content}`}
+              style={{
+                position: "absolute",
```

Comment:
let's extract this as a styled component instead of using inline style, so it's more readable 🙏